packages/data/addon/adapters/facts/bard.ts

```diff
@@ -151,11 +152,25 @@ export default class BardFactsAdapter extends EmberObject implements NaviFactAda
       );
     }
 
-    let [start, end] = timeFilter.values as string[];
-
-    const endMoment = end ? moment.utc(end) : undefined;
-    if (endMoment?.isValid()) {
-      end = endMoment.add(1, getPeriodForGrain(filterGrain)).toISOString();
+    let start: string, end: string;
+    if (timeFilter.operator === 'bet') {
+      [start, end] = timeFilter.values as string[];
+      // Add 1 period to convert [inclusive, inclusive] to [inclusive, exclusive) format
+      const endMoment = end ? moment.utc(end) : undefined;
+      if (endMoment?.isValid()) {
+        end = endMoment.add(1, getPeriodForGrain(filterGrain)).toISOString();
+      }
+    } else if (timeFilter.operator === 'gte') {
+      [start] = timeFilter.values as string[];
```

Comment:
Should we assert that value stored in `values` is an valid moment for `gte` & `lte`?